openmetadata-ui/src/main/resources/ui/src/components/CreateUser/CreateUser.component.tsx

```diff
@@ -0,0 +1,330 @@
+  };
+
+  /**
+   * Validate if required value is provided or not.
+   * @returns boolean
+   */
+
+  const validateForm = () => {
+    const errMsg = cloneDeep(showErrorMsg);
+    if (isEmpty(email)) {
+      errMsg.email = true;
+    } else {
+      errMsg.validEmail = !validEmailRegEx.test(email);
+    }
+    setShowErrorMsg(errMsg);
+
+    return !Object.values(errMsg).includes(true);
+  };
+
+  /**
+   * Form submit handler
+   */
+
+  const handleSave = () => {
+    const validRole = selectedRoles.filter(
+      (id) => !isUndefined(id)
+    ) as string[];
+    const validTeam = selectedTeams.filter(
+      (id) => !isUndefined(id)
+    ) as string[];
+    if (validateForm()) {
+      const userProfile: CreateUserSchema = {
+        description: markdownRef.current?.getEditorContent() || undefined,
+        name: email.split('@')[0],
+        roles: validRole.length ? validRole : undefined,
+        teams: validTeam.length ? validTeam : undefined,
+        email: email,
+        isAdmin: isAdmin,
+        isBot: isBot,
+      };
+      onSave(userProfile);
+    }
+  };
+
+  /**
+   * Dynamic button provided as per its state, useful for micro interaction
+   * @returns Button
+   */
+
+  const getSaveButton = () => {
+    return allowAccess ? (
+      <>
+        {saveState === 'waiting' ? (
+          <Button
+            disabled
+            className="tw-w-16 tw-h-10 disabled:tw-opacity-100"
+            size="regular"
+            theme="primary"
+            variant="contained">
+            <Loader size="small" type="white" />
+          </Button>
+        ) : saveState === 'success' ? (
+          <Button
+            disabled
+            className="tw-w-16 tw-h-10 disabled:tw-opacity-100"
+            size="regular"
+            theme="primary"
+            variant="contained">
+            <FontAwesomeIcon icon="check" />
+          </Button>
+        ) : (
+          <Button
+            className={classNames('tw-w-16 tw-h-10', {
+              'tw-opacity-40': !allowAccess,
+            })}
+            data-testid="save-user"
+            size="regular"
+            theme="primary"
+            variant="contained"
+            onClick={handleSave}>
+            Create
+          </Button>
+        )}
+      </>
+    ) : null;
+  };
+
+  return (
+    <PageLayout classes="tw-max-w-full-hd tw-h-full tw-bg-white tw-py-4">
+      <h6 className="tw-heading tw-text-base">Create User</h6>
+      <Field>
+        <label className="tw-block tw-form-label tw-mb-0" htmlFor="email">
+          {requiredField('Email:')}
+        </label>
+        <input
+          className="tw-form-inputs tw-px-3 tw-py-1"
+          data-testid="email"
+          id="email"
+          name="email"
+          placeholder="email"
+          type="text"
+          value={email}
+          onChange={handleValidation}
+        />
+        {showErrorMsg.email && errorMsg('Email is required.')}
+        {showErrorMsg.validEmail && errorMsg('Email is invalid.')}
```

Comment:
This should be like,
```
{showErrorMsg.email ? errorMsg('Email is required.') 
 : showErrorMsg.validEmail ? errorMsg('Email is invalid.') : null}
```

Reply:
Also error messages should be from `en.ts`, under json key `form-error-messages` similar to `api-error-messages`